lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -46,26 +46,29 @@ export class EvaluationClientWrapper {
       const credentials: VerifiableCredential[] = matches.map(
         (e) => jp.nodes(this._client.verifiableCredential, e)[0].value
       );
+      const areRequiredCredentialsPresent = this.determineAreRequiredCredentialsPresent(matchSubmissionRequirements);
```

Comment:
Call can be moved below the If-else of submissionrequirements presence check to avoid redundancy